src/index.html

```diff
@@ -11,6 +11,47 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      method="post"
+      action="http://google.pl"
+      data-qa="big"
+    >
+      <label class="label" for="search">
+        <img
```

Comment:
This image should be nested inside label tag, as the icon there is the label for the input